routes/admin.js

```diff
@@ -139,59 +136,29 @@ router.get('/applications', function (req, res, next) {
     //       sorting cannot be done sensibly.
     //
     // Typical request:
-    // 
+    //
     //       GET /applications?embed=1&filter=...&order_by=name%20ASC&offset=0&limit=20
-    utils.getFromAsync(req, res, '/applications', 200, function (err, appsResponse) {
+    var filterFields  = ['id','name','ownerEmail'];
+    var appsUri = utils.makePagingUri(req, '/applications?embed=1',filterFields);
```

Comment:
Please always run a code formatter over your code; if using VS Code, it's just Alt+Shift+F (here: Missing space after comma...). <nitpick>